File: Aporte 1/Actividad y Guia #5/app/src/main/java/com/example/myapptareareproductormusical/MainActivity.java
package com.example.myapptareareproductormusical;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ArrayAdapter;
import android.widget.Spinner;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    Spinner spiner;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        spiner = (Spinner) findViewById(R.id.spinner);
        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(MainActivity.this,R.array.generos, android.R.layout.simple_list_item_1);
        spiner.setAdapter(adapter);
    }

    public void Seleccion(View view) {
        String genero =  (spiner.getSelectedItem().toString());
        Bundle extras = new Bundle();
        extras.putString("Genero",String.valueOf(genero));
        Intent intent = new Intent(MainActivity.this, GenerosActivity.class);
        intent.putExtras(extras);
        startActivity(intent);
    }

    public void salir(View view) {
            finish();

    }
}

File: Aporte 1/AppinKotlin1/app/src/main/java/com/example/appinkotlin/MainActivity.kt
package com.example.appinkotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnCalculadora.setOnClickListener {
            val intent0:Intent = Intent(this, Calculadora::class.java)
            startActivity(intent0)
        }

        btnCronometro.setOnClickListener{
            val intent1:Intent = Intent(this, Cronometro::class.java)
            startActivity(intent1)
        }
        btnSalir.setOnClickListener {
            finish()
        }
    }

}